install.sh

```diff
@@ -317,6 +317,17 @@ EOABORT
 )"
 fi
 
+UNAME_MACHINE="$(uname -m)"
+if [[ "$UNAME_MACHINE" == "arm64" ]]; then
+  abort "$(cat <<EOABORT
+Homebrew is not (yet) supported on ARM processors!
+Rerun the Homebrew installer under Rosetta 2.
+EOABORT
+)"
+elif [[ "$UNAME_MACHINE" != "x86_64" ]]; then
+  abort "Homebrew is only supported on Intel processors!"
+fi
```

Comment:
Is this the case on Linux as well? ARM is allowed by `brew` on Linux.

Reply:
It's not supported on Linux also, correct.